protocol/0043-MKTL-market_lifecycle.md

```diff
@@ -316,15 +316,15 @@ The market state is `active`.
 When this is approved and enacted the market state is `closed`.
 Parties that had open positions see settlement cash-flows happen to settle positions.
 Margin account balances are transferred to the general account.
-After `market.liquidity.successorLaunchWindowLength` has passed since market settlement, any insurance pool balance is [redistributed](./0015-INSR-market_insurance_pool_collateral.md) equally between the global insurance pool and the insurance pools of the remaining active markets using the same settlement asset.
+After `market.liquidity.successorLaunchWindowLength` has passed since market settlement, any insurance pool balance is [redistributed](./0015-INSR-market_insurance_pool_collateral.md) into the global insurance pool using the same settlement asset.
 
-### Market never leaves opening auction, trading terminated triggered, market cancelled (<a name="0043-MKTL-003" href="#0043-MKTL-003">0043-MKTL-003</a>)
+### Market never leaves opening auction, trading terminated triggered, market cancelled (<a name="0043-MKTL-010" href="#0043-MKTL-010">0043-MKTL-010</a>)
 
 1. A market is proposed, approved by governance process and enters the opening auction (Pending state).
 1. Trading terminated data source triggers before the market leaves the opening auction (so market never left Pending state so far).
 1. All orders should be cancelled and collateral returned to respective parties general account for the relevant asset.
 1. All LP commitments should be cancelled and their bond returned to the general account for the relevant asset.
-1. After `market.liquidity.successorLaunchWindowLength` has elapsed since market cancellation, any insurance pool balance should be [redistributed](./0015-INSR-market_insurance_pool_collateral.md) equally between the global insurance pool and the insurance pools of the remaining active markets using the same settlement asset.
+1. After `market.liquidity.successorLaunchWindowLength` has elapsed since market cancellation, any insurance pool balance should be [redistributed](./0015-INSR-market_insurance_pool_collateral.md) into the global insurance pool using the same settlement asset.
```

Comment:
```suggestion
1. After `market.liquidity.successorLaunchWindowLength` has elapsed since market cancellation, any insurance pool balance should be transferred into the global insurance pool using the same settlement asset.
```